waspc/src/Wasp/Generator/SdkGenerator/Server/OperationsGenerator.hs

```diff
@@ -129,14 +141,15 @@ serverOperationsDirInSdkRootDir =
     (AS.Operation.QueryOp _ _) -> [reldir|queries|]
     (AS.Operation.ActionOp _ _) -> [reldir|actions|]
 
-getOperationTmplData :: AS.Operation.Operation -> Aeson.Value
-getOperationTmplData operation =
+getOperationTmplData :: Bool -> AS.Operation.Operation -> Aeson.Value
+getOperationTmplData isAuthEnabledGlobally operation  =
```

Comment:
Dragging `isAuthEnabledGlobally` all the way down seems a little silly.

Querying the app spec for an Operation would ideally tell us whether it's authenticated or not, but that's not how the AppSpec works.

@Martinsos am I missing something?